scripts/release_npm.sh

```diff
@@ -35,6 +35,12 @@ fi
 CMD_UPDATE_VERSION="lerna version ${PACKAGE_VERSION} --yes --exact --force-publish --no-git-tag-version --no-push"
 CMD_PUBLISH_PACKAGES="lerna publish ${PACKAGE_VERSION} ${TAG} --yes --exact --force-publish --no-git-tag-version --no-push --no-verify-access --no-verify-registry"
```

Comment:
No specifying the `PACKAGE_VERSION` here will cause publish command to fail because of un-staged files.